The PAGE_SIZE constant is not even used in the
application but conflicts with musl-dev

diff --git a/RKDevice.h b/RKDevice.h
index 2d81ae9..d608e8f 100644
--- a/RKDevice.h
+++ b/RKDevice.h
@@ -6,7 +6,6 @@
 #include "DefineHeader.h"
 
 #define SECTOR_SIZE 512
-#define PAGE_SIZE 2048
 #define SPARE_SIZE 16
 #define CHIPINFO_LEN 16
 #define IDBLOCK_TOP 50
@@ -128,4 +127,4 @@ protected:
 	char  m_layerName[32];
 };
 
-#endif
\ No newline at end of file
+#endif
